core/Cargo.toml

```
@@ -119,6 +119,7 @@ solana-net-utils = { workspace = true }
 solana-nonce = { workspace = true }
 solana-nonce-account = { workspace = true }
 solana-packet = { workspace = true }
+solana-patches = { path = "../patches" }
```

Just use
```suggestion
solana-patches = { workspace = true }
```
since we already define it as a workspace level dependency